2017-12-03T21:00:45.948+0900 I CONTROL  [initandlisten] MongoDB starting : pid=9558 port=47018 dbpath=/data/shard1/rs1 64-bit host=kris-ThinkPad-X1-Carbon-5th
2017-12-03T21:00:45.948+0900 I CONTROL  [initandlisten] db version v3.4.10
2017-12-03T21:00:45.948+0900 I CONTROL  [initandlisten] git version: 078f28920cb24de0dd479b5ea6c66c644f6326e9
2017-12-03T21:00:45.948+0900 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
2017-12-03T21:00:45.948+0900 I CONTROL  [initandlisten] allocator: tcmalloc
2017-12-03T21:00:45.948+0900 I CONTROL  [initandlisten] modules: none
2017-12-03T21:00:45.948+0900 I CONTROL  [initandlisten] build environment:
2017-12-03T21:00:45.948+0900 I CONTROL  [initandlisten]     distmod: ubuntu1604
2017-12-03T21:00:45.948+0900 I CONTROL  [initandlisten]     distarch: x86_64
2017-12-03T21:00:45.948+0900 I CONTROL  [initandlisten]     target_arch: x86_64
2017-12-03T21:00:45.948+0900 I CONTROL  [initandlisten] options: { net: { port: 47018 }, processManagement: { fork: true }, replication: { replSet: "s1" }, sharding: { clusterRole: "shardsvr" }, storage: { dbPath: "/data/shard1/rs1" }, systemLog: { destination: "file", path: "s1-r1.log" } }
2017-12-03T21:00:45.970+0900 I STORAGE  [initandlisten] 
2017-12-03T21:00:45.970+0900 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
2017-12-03T21:00:45.970+0900 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
2017-12-03T21:00:45.970+0900 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7269M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),checkpoint=(wait=60,log_size=2GB),statistics_log=(wait=0),
2017-12-03T21:00:46.050+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:46.051+0900 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2017-12-03T21:00:46.051+0900 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2017-12-03T21:00:46.051+0900 I CONTROL  [initandlisten] ** WARNING: You are running this process as the root user, which is not recommended.
2017-12-03T21:00:46.051+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:46.051+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:46.051+0900 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/enabled is 'always'.
2017-12-03T21:00:46.051+0900 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
2017-12-03T21:00:46.051+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:46.052+0900 W SHARDING [initandlisten] Started with --shardsvr, but no shardIdentity document was found on disk in admin.system.version. This most likely means this server has not yet been added to a sharded cluster.
2017-12-03T21:00:46.079+0900 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/shard1/rs1/diagnostic.data'
2017-12-03T21:00:46.107+0900 I REPL     [initandlisten] Did not find local voted for document at startup.
2017-12-03T21:00:46.107+0900 I REPL     [initandlisten] Did not find local replica set configuration document at startup;  NoMatchingDocument: Did not find replica set configuration document in local.system.replset
2017-12-03T21:00:46.108+0900 I NETWORK  [thread1] waiting for connections on port 47018
2017-12-03T21:00:51.353+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43686 #1 (1 connection now open)
2017-12-03T21:00:51.354+0900 I -        [conn1] end connection 127.0.0.1:43686 (1 connection now open)
2017-12-03T21:00:51.354+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43690 #2 (1 connection now open)
2017-12-03T21:00:51.354+0900 I NETWORK  [conn2] received client metadata from 127.0.0.1:43690 conn2: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:00:51.355+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47017
2017-12-03T21:00:51.356+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47017, took 1ms (1 connections now open to localhost:47017)
2017-12-03T21:00:53.357+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43704 #3 (2 connections now open)
2017-12-03T21:00:53.357+0900 I -        [conn3] end connection 127.0.0.1:43704 (2 connections now open)
2017-12-03T21:00:53.402+0900 I REPL     [replExecDBWorker-0] Starting replication storage threads
2017-12-03T21:00:53.473+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43708 #4 (2 connections now open)
2017-12-03T21:00:53.473+0900 I NETWORK  [conn4] received client metadata from 127.0.0.1:43708 conn4: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:00:53.474+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47019
2017-12-03T21:00:53.476+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47019, took 1ms (1 connections now open to localhost:47019)
2017-12-03T21:00:53.478+0900 I REPL     [replication-0] Starting initial sync (attempt 1 of 10)
2017-12-03T21:00:53.478+0900 I REPL     [ReplicationExecutor] New replica set config in use: { _id: "s1", version: 1, protocolVersion: 1, members: [ { _id: 0, host: "localhost:47017", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 }, { _id: 1, host: "localhost:47018", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 }, { _id: 2, host: "localhost:47019", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 } ], settings: { chainingAllowed: true, heartbeatIntervalMillis: 2000, heartbeatTimeoutSecs: 10, electionTimeoutMillis: 10000, catchUpTimeoutMillis: 60000, getLastErrorModes: {}, getLastErrorDefaults: { w: 1, wtimeout: 0 }, replicaSetId: ObjectId('5a23e7737da32b8ce5ce759f') } }
2017-12-03T21:00:53.478+0900 I REPL     [ReplicationExecutor] This node is localhost:47018 in the config
2017-12-03T21:00:53.478+0900 I REPL     [ReplicationExecutor] transition to STARTUP2
2017-12-03T21:00:53.479+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state SECONDARY
2017-12-03T21:00:53.479+0900 I REPL     [ReplicationExecutor] Member localhost:47019 is now in state STARTUP2
2017-12-03T21:00:53.515+0900 I REPL     [replication-0] sync source candidate: localhost:47017
2017-12-03T21:00:53.515+0900 I STORAGE  [replication-0] dropAllDatabasesExceptLocal 1
2017-12-03T21:00:53.515+0900 I REPL     [replication-0] ******
2017-12-03T21:00:53.515+0900 I REPL     [replication-0] creating replication oplog of size: 6116MB...
2017-12-03T21:00:53.534+0900 I STORAGE  [replication-0] Starting WiredTigerRecordStoreThread local.oplog.rs
2017-12-03T21:00:53.534+0900 I STORAGE  [replication-0] The size storer reports that the oplog contains 0 records totaling to 0 bytes
2017-12-03T21:00:53.534+0900 I STORAGE  [replication-0] Scanning the oplog to determine where to place markers for truncation
2017-12-03T21:00:53.634+0900 I REPL     [replication-0] ******
2017-12-03T21:00:53.635+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:00:53.637+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (1 connections now open to localhost:47017)
2017-12-03T21:00:53.639+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:00:53.641+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (2 connections now open to localhost:47017)
2017-12-03T21:00:53.645+0900 I REPL     [replication-1] No need to apply operations. (currently at { : Timestamp 1512302451000|1 })
2017-12-03T21:00:53.646+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending connection to host localhost:47017 due to bad connection status; 1 connections to that host remain open
2017-12-03T21:00:53.646+0900 I REPL     [replication-1] Finished fetching oplog during initial sync: CallbackCanceled: Callback canceled. Last fetched optime and hash: { ts: Timestamp 1512302451000|1, t: -1 }[2865188406503243212]
2017-12-03T21:00:53.647+0900 I REPL     [replication-1] Initial sync attempt finishing up.
2017-12-03T21:00:53.647+0900 I REPL     [replication-1] Initial Sync Attempt Statistics: { failedInitialSyncAttempts: 0, maxFailedInitialSyncAttempts: 10, initialSyncStart: new Date(1512302453478), initialSyncAttempts: [], fetchedMissingDocs: 0, appliedOps: 0, initialSyncOplogStart: Timestamp 1512302451000|1, initialSyncOplogEnd: Timestamp 1512302451000|1, databases: { databasesCloned: 0 } }
2017-12-03T21:00:53.654+0900 I REPL     [replication-1] initial sync done; took 0s.
2017-12-03T21:00:53.654+0900 I REPL     [replication-1] Starting replication fetcher thread
2017-12-03T21:00:53.654+0900 I REPL     [replication-1] Starting replication applier thread
2017-12-03T21:00:53.654+0900 I REPL     [replication-1] Starting replication reporter thread
2017-12-03T21:00:53.654+0900 I REPL     [rsSync] transition to RECOVERING
2017-12-03T21:00:53.655+0900 I REPL     [rsBackgroundSync] could not find member to sync from
2017-12-03T21:00:53.656+0900 I REPL     [rsSync] transition to SECONDARY
2017-12-03T21:00:53.656+0900 I REPL     [ReplicationExecutor] Member localhost:47019 is now in state SECONDARY
2017-12-03T21:01:03.657+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state PRIMARY
2017-12-03T21:01:08.659+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:47019
2017-12-03T21:01:08.659+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-03T21:01:08.661+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 2ms (1 connections now open to localhost:47019)
2017-12-03T21:01:08.662+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-03T21:01:08.663+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 1ms (2 connections now open to localhost:47019)
2017-12-03T21:02:10.737+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43882 #5 (3 connections now open)
2017-12-03T21:02:10.737+0900 I NETWORK  [conn5] received client metadata from 127.0.0.1:43882 conn5: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.832+0900 I INDEX    [repl writer worker 1] build index on: admin.system.version properties: { v: 2, key: { version: 1 }, name: "incompatible_with_version_32", ns: "admin.system.version" }
2017-12-03T21:02:10.832+0900 I INDEX    [repl writer worker 1] 	 building index using bulk method; build may temporarily use up to 500 megabytes of RAM
2017-12-03T21:02:10.833+0900 I INDEX    [repl writer worker 1] build index done.  scanned 0 total records. 0 secs
2017-12-03T21:02:10.834+0900 I COMMAND  [repl writer worker 2] setting featureCompatibilityVersion to 3.4
2017-12-03T21:02:10.834+0900 I SHARDING [repl writer worker 7] initializing sharding state with: { _id: "shardIdentity", configsvrConnectionString: "csReplSet/localhost:57040,localhost:57041,localhost:57042", shardName: "s1", clusterId: ObjectId('5a23e786bf51d32cbcc946a4') }
2017-12-03T21:02:10.834+0900 I SHARDING [repl writer worker 7] first cluster operation detected, adding sharding hook to enable versioning and authentication to remote servers
2017-12-03T21:02:10.835+0900 I NETWORK  [repl writer worker 7] Starting new replica set monitor for csReplSet/localhost:57040,localhost:57041,localhost:57042
2017-12-03T21:02:10.836+0900 I SHARDING [thread2] creating distributed lock ping thread for process kris-ThinkPad-X1-Carbon-5th:47018:1512302530:4272451777790352026 (sleeping for 30000ms)
2017-12-03T21:02:10.837+0900 I NETWORK  [replSetDistLockPinger] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-03T21:02:10.837+0900 I NETWORK  [shard registry reload] Successfully connected to localhost:57040 (1 connections now open to localhost:57040 with a 5 second timeout)
2017-12-03T21:02:10.837+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-03T21:02:10.838+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57042 (1 connections now open to localhost:57042 with a 5 second timeout)
2017-12-03T21:02:10.843+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-03T21:02:10.843+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 6ms (2 connections now open to localhost:57040)
2017-12-03T21:02:10.844+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 5ms (2 connections now open to localhost:57040)
2017-12-03T21:02:10.845+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s0/localhost:37017,localhost:37018,localhost:37019
2017-12-03T21:02:10.845+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s1/localhost:47017,localhost:47018,localhost:47019
2017-12-03T21:02:10.850+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: LockStateChangeFailed: findAndModify query predicate didn't match any lock document
2017-12-03T21:02:10.851+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43954 #6 (4 connections now open)
2017-12-03T21:02:10.851+0900 I NETWORK  [conn6] received client metadata from 127.0.0.1:43954 conn6: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.854+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-03T21:02:10.855+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37019 (1 connections now open to localhost:37019 with a 5 second timeout)
2017-12-03T21:02:10.856+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43958 #7 (5 connections now open)
2017-12-03T21:02:10.856+0900 I NETWORK  [conn7] received client metadata from 127.0.0.1:43958 conn7: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.856+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37018 (1 connections now open to localhost:37018 with a 5 second timeout)
2017-12-03T21:02:10.880+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43964 #8 (6 connections now open)
2017-12-03T21:02:10.880+0900 I NETWORK  [conn8] received client metadata from 127.0.0.1:43964 conn8: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.880+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47017 (1 connections now open to localhost:47017 with a 5 second timeout)
2017-12-03T21:02:10.881+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47019 (1 connections now open to localhost:47019 with a 5 second timeout)
2017-12-03T21:02:10.881+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43970 #9 (7 connections now open)
2017-12-03T21:02:10.881+0900 I NETWORK  [conn9] received client metadata from 127.0.0.1:43970 conn9: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.881+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47018 (1 connections now open to localhost:47018 with a 5 second timeout)
2017-12-03T21:02:10.930+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:44024 #10 (8 connections now open)
2017-12-03T21:02:10.930+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:44026 #11 (9 connections now open)
2017-12-03T21:02:10.930+0900 I NETWORK  [conn10] received client metadata from 127.0.0.1:44024 conn10: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.930+0900 I NETWORK  [conn11] received client metadata from 127.0.0.1:44026 conn11: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:11.015+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:44048 #12 (10 connections now open)
2017-12-03T21:02:11.015+0900 I NETWORK  [conn12] received client metadata from 127.0.0.1:44048 conn12: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:11.202+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:44064 #13 (11 connections now open)
2017-12-03T21:02:11.202+0900 I NETWORK  [conn13] received client metadata from 127.0.0.1:44064 conn13: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:27.452+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:44096 #14 (12 connections now open)
2017-12-03T21:02:27.453+0900 I NETWORK  [conn14] received client metadata from 127.0.0.1:44096 conn14: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:27.609+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:44114 #15 (13 connections now open)
2017-12-03T21:02:27.609+0900 I NETWORK  [conn15] received client metadata from 127.0.0.1:44114 conn15: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.676+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:44140 #16 (14 connections now open)
2017-12-03T21:02:40.677+0900 I NETWORK  [conn16] received client metadata from 127.0.0.1:44140 conn16: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.679+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:44146 #17 (15 connections now open)
2017-12-03T21:02:40.680+0900 I NETWORK  [conn17] received client metadata from 127.0.0.1:44146 conn17: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.845+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T21:02:40.847+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-03T21:02:40.848+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s2/localhost:57017,localhost:57018,localhost:57019
2017-12-03T21:02:40.849+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-03T21:02:40.852+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57018 (1 connections now open to localhost:57018 with a 5 second timeout)
2017-12-03T21:02:40.855+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57019 (1 connections now open to localhost:57019 with a 5 second timeout)
2017-12-03T21:03:10.845+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Ending idle connection to host localhost:57040 because the pool meets constraints; 1 connections to that host remain open
2017-12-03T21:03:10.848+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T21:03:10.850+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-03T21:09:40.866+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T21:09:40.867+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-03T21:55:10.958+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T21:55:10.960+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-03T22:03:40.979+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T22:03:40.982+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 3ms (1 connections now open to localhost:57041)
2017-12-03T22:41:11.077+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T22:41:11.078+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-03T23:17:41.152+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T23:17:41.154+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-03T23:32:11.188+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T23:32:11.190+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T00:06:41.267+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T00:06:41.268+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-04T00:25:41.311+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T00:25:41.313+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T00:30:11.323+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T00:30:11.324+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T00:42:41.356+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T00:42:41.358+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T00:53:11.383+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T00:53:11.384+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T01:03:09.904+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-04T01:03:09.906+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 2ms (3 connections now open to localhost:47019)
2017-12-04T01:04:09.905+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47019 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T01:25:41.467+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T01:25:41.469+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T01:46:41.520+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T01:46:41.524+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-04T01:59:11.555+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T01:59:11.556+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T02:04:41.568+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T02:04:41.570+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T02:20:11.606+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T02:20:11.607+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-04T02:35:11.644+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T02:35:11.646+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T03:10:11.733+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T03:10:11.734+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T04:23:41.920+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T04:23:41.922+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T04:40:11.962+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T04:40:11.964+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T05:21:12.078+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T05:21:12.080+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T05:56:42.169+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T05:56:42.171+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T06:51:42.314+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T06:51:42.315+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T07:26:12.406+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T07:26:12.408+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T07:28:42.413+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T07:28:42.415+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T15:41:54.169+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47019 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T15:41:54.176+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending connection to host localhost:47019 due to bad connection status; 0 connections to that host remain open
2017-12-04T15:41:54.185+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-04T15:41:54.180+0900 I REPL     [ReplicationExecutor] Starting an election, since we've seen no PRIMARY in the past 10000ms
2017-12-04T15:41:54.180+0900 I REPL     [replication-2] Restarting oplog query due to error: ExceededTimeLimit: Operation timed out, request was RemoteCommand 126641 -- target:localhost:47019 db:local expDate:2017-12-04T08:14:55.761+0900 cmd:{ getMore: 34604852486, collection: "oplog.rs", maxTimeMS: 5000, term: 1, lastKnownCommittedOpTime: { ts: Timestamp 1512342825000|1, t: 1 } }. Last fetched optime (with hash): { ts: Timestamp 1512342825000|1, t: 1 }[5334871594228210341]. Restarts remaining: 3
2017-12-04T15:41:54.185+0900 I REPL     [replication-2] Scheduled new oplog query Fetcher source: localhost:47019 database: local query: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512342825000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 1, readConcern: { afterOpTime: { ts: Timestamp 1512342825000|1, t: 1 } } } query metadata: { $replData: 1, $oplogQueryData: 1, $ssm: { $secondaryOk: true } } active: 1 timeout: 65000ms shutting down?: 0 first: 1 firstCommandScheduler: RemoteCommandRetryScheduler request: RemoteCommand 126658 -- target:localhost:47019 db:local cmd:{ find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512342825000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 1, readConcern: { afterOpTime: { ts: Timestamp 1512342825000|1, t: 1 } } } active: 1 callbackHandle.valid: 1 callbackHandle.cancelled: 0 attempt: 1 retryPolicy: RetryPolicyImpl maxAttempts: 1 maxTimeMillis: -1ms
2017-12-04T15:41:54.185+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-04T15:41:54.185+0900 I REPL     [ReplicationExecutor] conducting a dry run election to see if we could be elected
2017-12-04T15:41:54.187+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47017
2017-12-04T15:41:54.187+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47019
2017-12-04T15:41:54.197+0900 I REPL     [ReplicationExecutor] VoteRequester(term 1 dry run) received a yes vote from localhost:47019; response message: { term: 1, voteGranted: true, reason: "", ok: 1.0 }
2017-12-04T15:41:54.197+0900 I REPL     [ReplicationExecutor] dry election run succeeded, running for election
2017-12-04T15:41:54.202+0900 I REPL     [ReplicationExecutor] VoteRequester(term 2) received a no vote from localhost:47019 with reason "already voted for another candidate this term"; response message: { term: 2, voteGranted: false, reason: "already voted for another candidate this term", ok: 1.0 }
2017-12-04T15:41:54.205+0900 I REPL     [ReplicationExecutor] VoteRequester(term 2) received a no vote from localhost:47017 with reason "candidate's data is staler than mine"; response message: { term: 2, voteGranted: false, reason: "candidate's data is staler than mine", ok: 1.0 }
2017-12-04T15:41:54.205+0900 I REPL     [ReplicationExecutor] not becoming primary, we received insufficient votes
2017-12-04T15:41:54.210+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 25ms (2 connections now open to localhost:47019)
2017-12-04T15:41:54.210+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 25ms (2 connections now open to localhost:47019)
2017-12-04T15:41:54.212+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57040 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T15:41:54.212+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T15:41:54.212+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47019, took 26ms (2 connections now open to localhost:47019)
2017-12-04T15:41:54.212+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47017, took 26ms (2 connections now open to localhost:47017)
2017-12-04T15:41:54.212+0900 W REPL     [rsBackgroundSync] Fetcher stopped querying remote oplog with error: InvalidSyncSource: Sync source's last applied OpTime { ts: Timestamp 1512342825000|1, t: 1 } is not greater than our last fetched OpTime { ts: Timestamp 1512342825000|1, t: 1 }. Choosing new sync source.
2017-12-04T15:41:54.212+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:47017
2017-12-04T15:41:54.213+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T15:41:54.217+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 5ms (1 connections now open to localhost:47017)
2017-12-04T15:41:54.240+0900 I REPL     [SyncSourceFeedback] SyncSourceFeedback error sending update to localhost:47019: InvalidSyncSource: Sync source changed from localhost:47019 to localhost:47017
2017-12-04T15:41:54.242+0900 I REPL     [replication-0] Choosing new sync source because our current sync source, localhost:47017, has an OpTime ({ ts: Timestamp 1512369714000|1, t: 1 }) which is not ahead of ours ({ ts: Timestamp 1512369714000|1, t: 1 }), it does not have a sync source, and it's not the primary (sync source does not know the primary)
2017-12-04T15:41:54.242+0900 I REPL     [replication-0] Canceling oplog query due to OplogQueryMetadata. We have to choose a new sync source. Current source: localhost:47017, OpTime { ts: Timestamp 1512369714000|1, t: 1 }, its sync source index:-1
2017-12-04T15:41:54.242+0900 W REPL     [rsBackgroundSync] Fetcher stopped querying remote oplog with error: InvalidSyncSource: sync source localhost:47017 (config version: 1; last applied optime: { ts: Timestamp 1512369714000|1, t: 1 }; sync source index: -1; primary index: -1) is no longer valid
2017-12-04T15:41:54.242+0900 I REPL     [rsBackgroundSync] could not find member to sync from
2017-12-04T15:41:54.242+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state SECONDARY
2017-12-04T15:41:54.250+0900 I REPL     [SyncSourceFeedback] SyncSourceFeedback error sending update to localhost:47017: InvalidSyncSource: Sync source was cleared. Was localhost:47017
2017-12-04T15:42:04.639+0900 I -        [conn4] end connection 127.0.0.1:43708 (15 connections now open)
2017-12-04T15:42:04.640+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48070 #18 (15 connections now open)
2017-12-04T15:42:04.641+0900 I NETWORK  [conn18] received client metadata from 127.0.0.1:48070 conn18: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:42:09.243+0900 I REPL     [ReplicationExecutor] Member localhost:47019 is now in state PRIMARY
2017-12-04T15:42:09.245+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:47019
2017-12-04T15:42:10.465+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T15:42:10.466+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T15:42:10.837+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:57017)
2017-12-04T15:42:10.839+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-04T15:42:10.840+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:57040)
2017-12-04T15:42:10.842+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57040 (1 connections now open to localhost:57040 with a 5 second timeout)
2017-12-04T15:42:10.844+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:37017)
2017-12-04T15:42:10.845+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-04T15:42:10.847+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:47017)
2017-12-04T15:42:10.848+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47017 (1 connections now open to localhost:47017 with a 5 second timeout)
2017-12-04T15:42:24.213+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T15:42:24.216+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 4ms (1 connections now open to localhost:57041)
2017-12-04T15:42:54.202+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:47019 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T15:42:54.205+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T15:57:11.239+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T15:57:11.243+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 4ms (1 connections now open to localhost:57040)
2017-12-04T16:12:11.267+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T16:12:11.269+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-04T16:45:25.216+0900 I REPL     [ReplicationExecutor] Member localhost:47019 is now in state SECONDARY
2017-12-04T16:45:25.268+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57041 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T16:45:25.268+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T16:45:30.218+0900 I REPL     [replication-0] Choosing new sync source because our current sync source, localhost:47019, has an OpTime ({ ts: Timestamp 1512372452000|1, t: 3 }) which is not ahead of ours ({ ts: Timestamp 1512372452000|1, t: 3 }), it does not have a sync source, and it's not the primary (sync source does not know the primary)
2017-12-04T16:45:30.218+0900 I REPL     [replication-0] Canceling oplog query due to OplogQueryMetadata. We have to choose a new sync source. Current source: localhost:47019, OpTime { ts: Timestamp 1512372452000|1, t: 3 }, its sync source index:-1
2017-12-04T16:45:30.218+0900 W REPL     [rsBackgroundSync] Fetcher stopped querying remote oplog with error: InvalidSyncSource: sync source localhost:47019 (config version: 1; last applied optime: { ts: Timestamp 1512372452000|1, t: 3 }; sync source index: -1; primary index: -1) is no longer valid
2017-12-04T16:45:30.218+0900 I REPL     [rsBackgroundSync] could not find member to sync from
2017-12-04T16:45:30.219+0900 I REPL     [SyncSourceFeedback] SyncSourceFeedback error sending update to localhost:47019: InvalidSyncSource: Sync source was cleared. Was localhost:47019
2017-12-04T16:45:31.892+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T16:45:31.894+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T16:45:32.303+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 7 secs, remote host 127.0.0.1:57019)
2017-12-04T16:45:32.305+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57019 (1 connections now open to localhost:57019 with a 5 second timeout)
2017-12-04T16:45:32.306+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 7 secs, remote host 127.0.0.1:57041)
2017-12-04T16:45:32.308+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-04T16:45:32.309+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 7 secs, remote host 127.0.0.1:37018)
2017-12-04T16:45:32.310+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37018 (1 connections now open to localhost:37018 with a 5 second timeout)
2017-12-04T16:45:32.311+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 7 secs, remote host 127.0.0.1:47019)
2017-12-04T16:45:32.313+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47019 (1 connections now open to localhost:47019 with a 5 second timeout)
2017-12-04T16:45:32.314+0900 W NETWORK  [ReplicaSetMonitor-TaskExecutor-0] No primary detected for set s1
2017-12-04T16:45:35.790+0900 I REPL     [ReplicationExecutor] Starting an election, since we've seen no PRIMARY in the past 10000ms
2017-12-04T16:45:35.790+0900 I REPL     [ReplicationExecutor] conducting a dry run election to see if we could be elected
2017-12-04T16:45:35.791+0900 I REPL     [ReplicationExecutor] VoteRequester(term 3 dry run) received a yes vote from localhost:47017; response message: { term: 3, voteGranted: true, reason: "", ok: 1.0 }
2017-12-04T16:45:35.791+0900 I REPL     [ReplicationExecutor] dry election run succeeded, running for election
2017-12-04T16:45:35.797+0900 I REPL     [ReplicationExecutor] VoteRequester(term 4) received a yes vote from localhost:47017; response message: { term: 4, voteGranted: true, reason: "", ok: 1.0 }
2017-12-04T16:45:35.797+0900 I REPL     [ReplicationExecutor] election succeeded, assuming primary role in term 4
2017-12-04T16:45:35.797+0900 I REPL     [ReplicationExecutor] transition to PRIMARY
2017-12-04T16:45:35.797+0900 I REPL     [ReplicationExecutor] Entering primary catch-up mode.
2017-12-04T16:45:35.797+0900 I REPL     [ReplicationExecutor] Caught up to the latest optime known via heartbeats after becoming primary.
2017-12-04T16:45:35.798+0900 I REPL     [ReplicationExecutor] Exited primary catch-up mode.
2017-12-04T16:45:37.218+0900 I REPL     [rsSync] transition to primary complete; database writes are now permitted
2017-12-04T16:45:40.194+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37890 #19 (16 connections now open)
2017-12-04T16:45:40.195+0900 I NETWORK  [conn19] received client metadata from 127.0.0.1:37890 conn19: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:40.197+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37892 #20 (17 connections now open)
2017-12-04T16:45:40.197+0900 I NETWORK  [conn20] received client metadata from 127.0.0.1:37892 conn20: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:40.256+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37894 #21 (18 connections now open)
2017-12-04T16:45:40.256+0900 I NETWORK  [conn21] received client metadata from 127.0.0.1:37894 conn21: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:40.258+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37896 #22 (19 connections now open)
2017-12-04T16:45:40.259+0900 I NETWORK  [conn22] received client metadata from 127.0.0.1:37896 conn22: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:51.230+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38014 #23 (20 connections now open)
2017-12-04T16:45:51.231+0900 I NETWORK  [conn23] received client metadata from 127.0.0.1:38014 conn23: { driver: { name: "NetworkInterfaceASIO-ShardRegistry", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:46:25.219+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47019 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T16:46:31.897+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T16:46:31.897+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 0ms (1 connections now open to localhost:57040)
2017-12-04T17:01:31.931+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T17:01:31.933+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T17:38:04.887+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36404 #24 (21 connections now open)
2017-12-04T17:38:04.888+0900 I NETWORK  [conn24] received client metadata from 127.0.0.1:36404 conn24: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T17:38:04.905+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36406 #25 (22 connections now open)
2017-12-04T17:38:04.905+0900 I NETWORK  [conn25] received client metadata from 127.0.0.1:36406 conn25: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-3", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T17:38:11.971+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36468 #26 (23 connections now open)
2017-12-04T17:38:11.972+0900 I NETWORK  [conn26] received client metadata from 127.0.0.1:36468 conn26: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-0", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T17:38:42.279+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36718 #27 (24 connections now open)
2017-12-04T17:38:42.280+0900 I NETWORK  [conn27] received client metadata from 127.0.0.1:36718 conn27: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-1", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T17:39:32.672+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37120 #28 (25 connections now open)
2017-12-04T17:39:32.672+0900 I NETWORK  [conn28] received client metadata from 127.0.0.1:37120 conn28: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-2", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T17:44:53.964+0900 I -        [conn26] end connection 127.0.0.1:36468 (25 connections now open)
2017-12-04T17:45:13.889+0900 I -        [conn27] end connection 127.0.0.1:36718 (24 connections now open)
2017-12-04T17:45:20.409+0900 I -        [conn28] end connection 127.0.0.1:37120 (23 connections now open)
2017-12-04T17:45:23.469+0900 I -        [conn25] end connection 127.0.0.1:36406 (22 connections now open)
2017-12-04T17:50:04.167+0900 I INDEX    [conn24] build index on: mongomart.item properties: { v: 2, key: { _fts: "text", _ftsx: 1 }, name: "title_text_slogan_text_description_text", ns: "mongomart.item", weights: { description: 1, slogan: 1, title: 1 }, default_language: "english", language_override: "language", textIndexVersion: 3 }
2017-12-04T17:50:04.168+0900 I INDEX    [conn24] 	 building index using bulk method; build may temporarily use up to 500 megabytes of RAM
2017-12-04T17:50:04.177+0900 I INDEX    [conn24] build index done.  scanned 23 total records. 0 secs
2017-12-04T17:50:40.646+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:42600 #29 (22 connections now open)
2017-12-04T17:50:40.646+0900 I NETWORK  [conn29] received client metadata from 127.0.0.1:42600 conn29: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-0", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T17:50:44.595+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:42636 #30 (23 connections now open)
2017-12-04T17:50:44.595+0900 I NETWORK  [conn30] received client metadata from 127.0.0.1:42636 conn30: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-1", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T17:50:57.819+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:42742 #31 (24 connections now open)
2017-12-04T17:50:57.820+0900 I NETWORK  [conn31] received client metadata from 127.0.0.1:42742 conn31: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-2", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T17:51:42.106+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43096 #32 (25 connections now open)
2017-12-04T17:51:42.106+0900 I NETWORK  [conn32] received client metadata from 127.0.0.1:43096 conn32: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-3", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T17:58:00.280+0900 I -        [conn30] end connection 127.0.0.1:42636 (25 connections now open)
2017-12-04T17:58:50.698+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:46612 #33 (25 connections now open)
2017-12-04T17:58:50.698+0900 I NETWORK  [conn33] received client metadata from 127.0.0.1:46612 conn33: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-1", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T18:02:33.071+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T18:02:33.072+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-04T18:03:45.708+0900 I -        [conn29] end connection 127.0.0.1:42600 (25 connections now open)
2017-12-04T18:03:50.699+0900 I -        [conn33] end connection 127.0.0.1:46612 (24 connections now open)
2017-12-04T18:04:20.686+0900 I -        [conn31] end connection 127.0.0.1:42742 (23 connections now open)
2017-12-04T18:04:29.801+0900 I -        [conn32] end connection 127.0.0.1:43096 (22 connections now open)
2017-12-04T18:05:02.174+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:49616 #34 (22 connections now open)
2017-12-04T18:05:02.174+0900 I NETWORK  [conn34] received client metadata from 127.0.0.1:49616 conn34: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-0", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T18:05:26.692+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:49810 #35 (23 connections now open)
2017-12-04T18:05:26.692+0900 I NETWORK  [conn35] received client metadata from 127.0.0.1:49810 conn35: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-1", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T18:06:12.899+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:50186 #36 (24 connections now open)
2017-12-04T18:06:12.900+0900 I NETWORK  [conn36] received client metadata from 127.0.0.1:50186 conn36: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-2", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T18:07:33.605+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:50860 #37 (25 connections now open)
2017-12-04T18:07:33.606+0900 I NETWORK  [conn37] received client metadata from 127.0.0.1:50860 conn37: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-3", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T18:11:12.902+0900 I -        [conn36] end connection 127.0.0.1:50186 (25 connections now open)
2017-12-04T18:12:28.823+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:53290 #38 (25 connections now open)
2017-12-04T18:12:28.823+0900 I NETWORK  [conn38] received client metadata from 127.0.0.1:53290 conn38: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-2", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T18:12:41.227+0900 I -        [conn35] end connection 127.0.0.1:49810 (25 connections now open)
2017-12-04T18:16:03.403+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:55086 #39 (25 connections now open)
2017-12-04T18:16:03.403+0900 I NETWORK  [conn39] received client metadata from 127.0.0.1:55086 conn39: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-1", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T18:27:35.272+0900 I -        [conn34] end connection 127.0.0.1:49616 (25 connections now open)
2017-12-04T18:27:50.878+0900 I -        [conn39] end connection 127.0.0.1:55086 (24 connections now open)
2017-12-04T18:27:57.952+0900 I -        [conn38] end connection 127.0.0.1:53290 (23 connections now open)
2017-12-04T18:28:04.414+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60928 #40 (23 connections now open)
2017-12-04T18:28:04.415+0900 I NETWORK  [conn40] received client metadata from 127.0.0.1:60928 conn40: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-0", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T18:28:04.433+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60930 #41 (24 connections now open)
2017-12-04T18:28:04.434+0900 I NETWORK  [conn41] received client metadata from 127.0.0.1:60930 conn41: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-1", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T18:28:13.294+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:32776 #42 (25 connections now open)
2017-12-04T18:28:13.295+0900 I NETWORK  [conn42] received client metadata from 127.0.0.1:32776 conn42: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T18:28:13.307+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:32778 #43 (26 connections now open)
2017-12-04T18:28:13.307+0900 I NETWORK  [conn43] received client metadata from 127.0.0.1:32778 conn43: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-2", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T18:33:28.578+0900 I -        [conn37] end connection 127.0.0.1:50860 (26 connections now open)
2017-12-04T18:33:28.579+0900 I -        [conn40] end connection 127.0.0.1:60928 (25 connections now open)
2017-12-04T18:34:06.443+0900 I -        [conn41] end connection 127.0.0.1:60930 (24 connections now open)
2017-12-04T18:34:06.446+0900 I -        [conn43] end connection 127.0.0.1:32778 (23 connections now open)
2017-12-04T18:52:42.444+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:44728 #44 (23 connections now open)
2017-12-04T18:52:42.444+0900 I NETWORK  [conn44] received client metadata from 127.0.0.1:44728 conn44: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T18:53:42.444+0900 I -        [conn22] end connection 127.0.0.1:37896 (23 connections now open)
2017-12-04T19:13:57.177+0900 I REPL     [ReplicationExecutor] can't see a majority of the set, relinquishing primary
2017-12-04T19:13:57.177+0900 I REPL     [ReplicationExecutor] Stepping down from primary in response to heartbeat
2017-12-04T19:13:57.177+0900 I COMMAND  [conn21] command local.oplog.rs command: getMore { getMore: 36122414565, collection: "oplog.rs", maxTimeMS: 5000, term: 4, lastKnownCommittedOpTime: { ts: Timestamp 1512382297000|1, t: 4 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512372452000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 4, readConcern: { afterOpTime: { ts: Timestamp 1512372452000|1, t: 3 } } } planSummary: COLLSCAN cursorid:36122414565 keysExamined:0 docsExamined:0 numYields:0 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 4 } }, Database: { acquireCount: { r: 2 } }, oplog: { acquireCount: { r: 2 } } } protocol:op_command 139700ms
2017-12-04T19:13:57.178+0900 I COMMAND  [conn20] command local.oplog.rs command: getMore { getMore: 37293789345, collection: "oplog.rs", maxTimeMS: 5000, term: 4, lastKnownCommittedOpTime: { ts: Timestamp 1512382297000|1, t: 4 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512372452000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 4, readConcern: { afterOpTime: { ts: Timestamp 1512372452000|1, t: 3 } } } planSummary: COLLSCAN cursorid:37293789345 keysExamined:0 docsExamined:0 numYields:0 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 4 } }, Database: { acquireCount: { r: 2 } }, oplog: { acquireCount: { r: 2 } } } protocol:op_command 139700ms
2017-12-04T19:13:57.179+0900 I REPL     [replExecDBWorker-0] transition to SECONDARY
2017-12-04T19:13:57.179+0900 I NETWORK  [replExecDBWorker-0] legacy transport layer closing all connections
2017-12-04T19:13:57.179+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 44
2017-12-04T19:13:57.179+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 23
2017-12-04T19:13:57.179+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 21
2017-12-04T19:13:57.179+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 20
2017-12-04T19:13:57.179+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 19
2017-12-04T19:13:57.179+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 18
2017-12-04T19:13:57.179+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 2
2017-12-04T19:13:57.179+0900 I -        [conn5] end connection 127.0.0.1:43882 (22 connections now open)
2017-12-04T19:13:57.179+0900 I -        [conn15] end connection 127.0.0.1:44114 (22 connections now open)
2017-12-04T19:13:57.179+0900 I -        [conn12] end connection 127.0.0.1:44048 (22 connections now open)
2017-12-04T19:13:57.180+0900 I -        [conn42] end connection 127.0.0.1:32776 (20 connections now open)
2017-12-04T19:13:57.180+0900 I -        [conn14] end connection 127.0.0.1:44096 (19 connections now open)
2017-12-04T19:13:57.180+0900 I -        [conn8] end connection 127.0.0.1:43964 (18 connections now open)
2017-12-04T19:13:57.181+0900 I -        [conn13] end connection 127.0.0.1:44064 (17 connections now open)
2017-12-04T19:13:57.179+0900 I -        [conn10] end connection 127.0.0.1:44024 (22 connections now open)
2017-12-04T19:13:57.182+0900 I -        [conn16] end connection 127.0.0.1:44140 (15 connections now open)
2017-12-04T19:13:57.179+0900 I -        [conn6] end connection 127.0.0.1:43954 (22 connections now open)
2017-12-04T19:13:57.182+0900 I -        [conn9] end connection 127.0.0.1:43970 (12 connections now open)
2017-12-04T19:13:57.182+0900 I REPL     [ReplicationExecutor] Member localhost:47019 is now in state SECONDARY
2017-12-04T19:13:57.182+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state SECONDARY
2017-12-04T19:13:57.182+0900 I -        [conn24] end connection 127.0.0.1:36404 (11 connections now open)
2017-12-04T19:13:57.182+0900 I -        [conn17] end connection 127.0.0.1:44146 (10 connections now open)
2017-12-04T19:13:57.183+0900 I -        [conn11] end connection 127.0.0.1:44026 (9 connections now open)
2017-12-04T19:13:57.184+0900 I -        [conn7] end connection 127.0.0.1:43958 (8 connections now open)
2017-12-04T19:13:57.244+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57042 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T19:13:57.244+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T19:13:59.688+0900 I -        [conn44] end connection 127.0.0.1:44728 (7 connections now open)
2017-12-04T19:14:07.571+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54018 #45 (7 connections now open)
2017-12-04T19:14:07.571+0900 I NETWORK  [conn45] received client metadata from 127.0.0.1:54018 conn45: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:07.674+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54026 #46 (8 connections now open)
2017-12-04T19:14:07.674+0900 I NETWORK  [conn46] received client metadata from 127.0.0.1:54026 conn46: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:08.082+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54034 #47 (9 connections now open)
2017-12-04T19:14:08.082+0900 I NETWORK  [conn47] received client metadata from 127.0.0.1:54034 conn47: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:12.184+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state PRIMARY
2017-12-04T19:14:13.180+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:47017
2017-12-04T19:14:13.180+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T19:14:13.181+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (1 connections now open to localhost:47017)
2017-12-04T19:14:13.182+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T19:14:13.183+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (2 connections now open to localhost:47017)
2017-12-04T19:14:20.595+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54176 #48 (10 connections now open)
2017-12-04T19:14:20.595+0900 I NETWORK  [conn48] received client metadata from 127.0.0.1:54176 conn48: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:20.604+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54186 #49 (11 connections now open)
2017-12-04T19:14:20.604+0900 I NETWORK  [conn49] received client metadata from 127.0.0.1:54186 conn49: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:20.608+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54190 #50 (12 connections now open)
2017-12-04T19:14:20.608+0900 I NETWORK  [conn50] received client metadata from 127.0.0.1:54190 conn50: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:20.615+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54200 #51 (13 connections now open)
2017-12-04T19:14:20.616+0900 I NETWORK  [conn51] received client metadata from 127.0.0.1:54200 conn51: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:20.924+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 23 secs, remote host 127.0.0.1:57042)
2017-12-04T19:14:20.926+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57042 (1 connections now open to localhost:57042 with a 5 second timeout)
2017-12-04T19:14:20.928+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 23 secs, remote host 127.0.0.1:57017)
2017-12-04T19:14:20.929+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-04T19:14:20.931+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54216 #52 (14 connections now open)
2017-12-04T19:14:20.931+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 23 secs, remote host 127.0.0.1:37017)
2017-12-04T19:14:20.933+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-04T19:14:20.934+0900 I NETWORK  [conn52] received client metadata from 127.0.0.1:54216 conn52: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:20.935+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 23 secs, remote host 127.0.0.1:47018)
2017-12-04T19:14:20.935+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54222 #53 (15 connections now open)
2017-12-04T19:14:20.936+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54224 #54 (16 connections now open)
2017-12-04T19:14:20.936+0900 I NETWORK  [conn53] received client metadata from 127.0.0.1:54222 conn53: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:20.936+0900 I NETWORK  [conn54] received client metadata from 127.0.0.1:54224 conn54: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:20.936+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47018 (1 connections now open to localhost:47018 with a 5 second timeout)
2017-12-04T19:14:20.946+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54232 #55 (17 connections now open)
2017-12-04T19:14:20.946+0900 I NETWORK  [conn55] received client metadata from 127.0.0.1:54232 conn55: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:20.947+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54236 #56 (18 connections now open)
2017-12-04T19:14:20.947+0900 I NETWORK  [conn56] received client metadata from 127.0.0.1:54236 conn56: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:20.949+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54242 #57 (19 connections now open)
2017-12-04T19:14:20.949+0900 I NETWORK  [conn57] received client metadata from 127.0.0.1:54242 conn57: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:14:27.245+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T19:14:27.247+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T19:14:57.251+0900 I -        [conn19] end connection 127.0.0.1:37890 (19 connections now open)
2017-12-04T19:16:35.081+0900 I -        [conn23] end connection 127.0.0.1:38014 (18 connections now open)
2017-12-04T19:18:57.239+0900 I -        [conn21] end connection 127.0.0.1:37894 (17 connections now open)
2017-12-04T19:19:02.200+0900 I -        [conn20] end connection 127.0.0.1:37892 (16 connections now open)
2017-12-04T19:28:50.483+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T19:28:50.485+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T19:35:20.501+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T19:35:20.502+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-04T20:27:51.317+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T20:27:51.321+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 4ms (1 connections now open to localhost:57040)
2017-12-04T21:44:51.501+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T21:44:51.503+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-04T22:20:21.595+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T22:20:21.599+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 5ms (1 connections now open to localhost:57040)
2017-12-04T22:31:51.624+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T22:31:51.627+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 3ms (1 connections now open to localhost:57042)
2017-12-04T22:55:54.639+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T22:55:54.640+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-04T22:56:54.640+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T23:02:51.700+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T23:02:51.702+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T23:25:21.759+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T23:25:21.761+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T23:44:21.809+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T23:44:21.811+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T00:03:21.860+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T00:03:21.864+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 4ms (1 connections now open to localhost:57040)
2017-12-05T00:40:21.950+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T00:40:21.953+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 3ms (1 connections now open to localhost:57042)
2017-12-05T00:51:21.976+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T00:51:21.977+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T01:26:22.058+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T01:26:22.059+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-05T01:33:52.072+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T01:33:52.074+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T01:46:52.098+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T01:46:52.100+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T01:56:52.120+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T01:56:52.122+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T02:59:22.272+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T02:59:22.276+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-05T03:31:52.348+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T03:31:52.349+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T03:34:22.354+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T03:34:22.355+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-05T04:06:52.434+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T04:06:52.435+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T04:34:22.503+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T04:34:22.504+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T04:57:22.561+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T04:57:22.562+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T05:16:22.610+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T05:16:22.611+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-05T06:15:22.732+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T06:15:22.733+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-05T06:53:52.807+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T06:53:52.809+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T07:44:52.916+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T07:44:52.917+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T07:46:52.923+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T07:46:52.923+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 0ms (1 connections now open to localhost:57042)
2017-12-05T07:51:52.936+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T07:51:52.937+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T15:30:53.179+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 1 connections to that host remain open
2017-12-05T15:30:53.179+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending connection to host localhost:47017 due to bad connection status; 0 connections to that host remain open
2017-12-05T15:30:53.183+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60154 #58 (16 connections now open)
2017-12-05T15:30:53.183+0900 I NETWORK  [conn58] received client metadata from 127.0.0.1:60154 conn58: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T15:30:53.219+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 26790 secs, remote host 127.0.0.1:37019)
2017-12-05T15:30:53.219+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 26790 secs, remote host 127.0.0.1:47017)
2017-12-05T15:30:53.219+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 26790 secs, remote host 127.0.0.1:57017)
2017-12-05T15:30:53.219+0900 I REPL     [replication-0] Restarting oplog query due to error: ExceededTimeLimit: Operation timed out, request was RemoteCommand 308641 -- target:localhost:47017 db:local expDate:2017-12-05T08:05:35.644+0900 cmd:{ getMore: 17418435668, collection: "oplog.rs", maxTimeMS: 5000, term: 5, lastKnownCommittedOpTime: { ts: Timestamp 1512428670000|1, t: 5 } }. Last fetched optime (with hash): { ts: Timestamp 1512428670000|1, t: 5 }[-3228870133416877210]. Restarts remaining: 3
2017-12-05T15:30:53.219+0900 I REPL     [replication-0] Scheduled new oplog query Fetcher source: localhost:47017 database: local query: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512428670000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 6, readConcern: { afterOpTime: { ts: Timestamp 1512428670000|1, t: 5 } } } query metadata: { $replData: 1, $oplogQueryData: 1, $ssm: { $secondaryOk: true } } active: 1 timeout: 65000ms shutting down?: 0 first: 1 firstCommandScheduler: RemoteCommandRetryScheduler request: RemoteCommand 308656 -- target:localhost:47017 db:local cmd:{ find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512428670000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 6, readConcern: { afterOpTime: { ts: Timestamp 1512428670000|1, t: 5 } } } active: 1 callbackHandle.valid: 1 callbackHandle.cancelled: 0 attempt: 1 retryPolicy: RetryPolicyImpl maxAttempts: 1 maxTimeMillis: -1ms
2017-12-05T15:30:53.220+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-05T15:30:53.220+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-05T15:30:53.243+0900 I REPL     [ReplicationExecutor] Member localhost:47019 is now in state PRIMARY
2017-12-05T15:30:53.246+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state SECONDARY
2017-12-05T15:30:53.246+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 27ms (2 connections now open to localhost:47017)
2017-12-05T15:30:53.246+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 27ms (2 connections now open to localhost:47017)
2017-12-05T15:30:53.249+0900 W REPL     [rsBackgroundSync] Fetcher stopped querying remote oplog with error: InvalidSyncSource: Sync source's last applied OpTime { ts: Timestamp 1512428670000|1, t: 5 } is not greater than our last fetched OpTime { ts: Timestamp 1512428670000|1, t: 5 }. Choosing new sync source.
2017-12-05T15:30:53.249+0900 I REPL     [rsBackgroundSync] could not find member to sync from
2017-12-05T15:30:53.253+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57041 as failed :: caused by :: NotMaster: Not primary while running findAndModify command on collection config.lockpings
2017-12-05T15:30:53.266+0900 I NETWORK  [replSetDistLockPinger] Detected bad connection created at 1512373532306671 microSec, clearing pool for localhost:57041 of 0 connections
2017-12-05T15:30:53.279+0900 I NETWORK  [replSetDistLockPinger] Dropping all pooled connections to localhost:57041(with timeout of 5 seconds)
2017-12-05T15:30:53.279+0900 I NETWORK  [replSetDistLockPinger] Ending connection to host localhost:57041(with timeout of 5 seconds) due to bad connection status; 0 connections to that host remain open
2017-12-05T15:30:53.283+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-05T15:30:53.783+0900 I NETWORK  [replSetDistLockPinger] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-05T15:30:53.784+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-05T15:30:58.247+0900 I REPL     [SyncSourceFeedback] SyncSourceFeedback error sending update to localhost:47017: InvalidSyncSource: Sync source was cleared. Was localhost:47017
2017-12-05T15:30:58.251+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:47019
2017-12-05T15:30:58.251+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-05T15:30:58.252+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 1ms (1 connections now open to localhost:47019)
2017-12-05T15:30:58.254+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-05T15:30:58.256+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 2ms (2 connections now open to localhost:47019)
2017-12-05T15:31:10.044+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-05T15:31:10.048+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37019 (1 connections now open to localhost:37019 with a 5 second timeout)
2017-12-05T15:31:10.051+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47017 (1 connections now open to localhost:47017 with a 5 second timeout)
2017-12-05T15:31:53.184+0900 I -        [conn18] end connection 127.0.0.1:48070 (16 connections now open)
2017-12-05T15:31:53.247+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 1 connections to that host remain open
2017-12-05T15:32:39.233+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T15:32:39.234+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-05T16:03:27.184+0900 I REPL     [ReplicationExecutor] Starting an election, since we've seen no PRIMARY in the past 10000ms
2017-12-05T16:03:27.184+0900 I REPL     [ReplicationExecutor] conducting a dry run election to see if we could be elected
2017-12-05T16:03:27.187+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47017
2017-12-05T16:03:27.187+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47019
2017-12-05T16:03:27.205+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47017, took 21ms (2 connections now open to localhost:47017)
2017-12-05T16:03:27.207+0900 I REPL     [ReplicationExecutor] VoteRequester(term 6 dry run) received a yes vote from localhost:47017; response message: { term: 6, voteGranted: true, reason: "", ok: 1.0 }
2017-12-05T16:03:27.207+0900 I REPL     [ReplicationExecutor] dry election run succeeded, running for election
2017-12-05T16:03:27.209+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47019
2017-12-05T16:03:27.211+0900 I REPL     [ReplicationExecutor] VoteRequester(term 7) received a yes vote from localhost:47017; response message: { term: 7, voteGranted: true, reason: "", ok: 1.0 }
2017-12-05T16:03:27.211+0900 I REPL     [ReplicationExecutor] election succeeded, assuming primary role in term 7
2017-12-05T16:03:27.211+0900 I REPL     [ReplicationExecutor] transition to PRIMARY
2017-12-05T16:03:27.211+0900 I REPL     [ReplicationExecutor] Entering primary catch-up mode.
2017-12-05T16:03:27.211+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47019
2017-12-05T16:03:27.211+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending connection to host localhost:47019 due to bad connection status; 3 connections to that host remain open
2017-12-05T16:03:27.223+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1517 secs, remote host 127.0.0.1:37017)
2017-12-05T16:03:27.223+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1517 secs, remote host 127.0.0.1:47019)
2017-12-05T16:03:27.223+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1517 secs, remote host 127.0.0.1:57018)
2017-12-05T16:03:27.223+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1517 secs, remote host 127.0.0.1:57040)
2017-12-05T16:03:27.238+0900 W REPL     [rsBackgroundSync] Fetcher stopped querying remote oplog with error: InvalidSyncSource: sync source localhost:47019 (config version: 1; last applied optime: { ts: Timestamp 1512455885000|1, t: 6 }; sync source index: -1; primary index: -1) is no longer valid
2017-12-05T16:03:27.249+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47019, took 40ms (3 connections now open to localhost:47019)
2017-12-05T16:03:27.249+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47019, took 38ms (3 connections now open to localhost:47019)
2017-12-05T16:03:27.249+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47019, took 65ms (3 connections now open to localhost:47019)
2017-12-05T16:03:27.249+0900 I REPL     [ReplicationExecutor] Member localhost:47019 is now in state SECONDARY
2017-12-05T16:03:27.249+0900 I REPL     [ReplicationExecutor] Caught up to the latest optime known via heartbeats after becoming primary.
2017-12-05T16:03:27.249+0900 I REPL     [ReplicationExecutor] Exited primary catch-up mode.
2017-12-05T16:03:27.258+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57040 as failed :: caused by :: NotMaster: not master
2017-12-05T16:03:27.259+0900 I NETWORK  [replSetDistLockPinger] Successfully connected to localhost:57040 (1 connections now open to localhost:57040 with a 5 second timeout)
2017-12-05T16:03:27.260+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-05T16:03:27.668+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47019 because the pool meets constraints; 1 connections to that host remain open
2017-12-05T16:03:27.668+0900 I REPL     [SyncSourceFeedback] SyncSourceFeedback error sending update to localhost:47019: InvalidSyncSource: Sync source was cleared. Was localhost:47019
2017-12-05T16:03:27.761+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-05T16:03:29.188+0900 I REPL     [rsSync] transition to primary complete; database writes are now permitted
2017-12-05T16:03:32.231+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35800 #59 (16 connections now open)
2017-12-05T16:03:32.232+0900 I NETWORK  [conn59] received client metadata from 127.0.0.1:35800 conn59: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:03:32.235+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35802 #60 (17 connections now open)
2017-12-05T16:03:32.235+0900 I NETWORK  [conn60] received client metadata from 127.0.0.1:35802 conn60: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:03:49.273+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35984 #61 (18 connections now open)
2017-12-05T16:03:49.274+0900 I NETWORK  [conn61] received client metadata from 127.0.0.1:35984 conn61: { driver: { name: "NetworkInterfaceASIO-ShardRegistry", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:03:52.539+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57018 (1 connections now open to localhost:57018 with a 5 second timeout)
2017-12-05T16:03:52.542+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-05T16:03:52.545+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47019 (1 connections now open to localhost:47019 with a 5 second timeout)
2017-12-05T16:04:27.205+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 1 connections to that host remain open
2017-12-05T16:04:27.249+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:47019 because the pool meets constraints; 2 connections to that host remain open
2017-12-05T16:04:27.249+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:47019 because the pool meets constraints; 1 connections to that host remain open
2017-12-05T16:04:46.202+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36510 #62 (19 connections now open)
2017-12-05T16:04:46.203+0900 I NETWORK  [conn62] received client metadata from 127.0.0.1:36510 conn62: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:04:54.506+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36578 #63 (20 connections now open)
2017-12-05T16:04:54.507+0900 I NETWORK  [conn63] received client metadata from 127.0.0.1:36578 conn63: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-0", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:05:23.012+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-05T16:05:23.014+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-05T16:05:59.203+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37106 #64 (21 connections now open)
2017-12-05T16:05:59.203+0900 I NETWORK  [conn64] received client metadata from 127.0.0.1:37106 conn64: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:06:59.203+0900 I -        [conn59] end connection 127.0.0.1:35800 (21 connections now open)
2017-12-05T16:07:59.527+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38118 #65 (21 connections now open)
2017-12-05T16:07:59.527+0900 I NETWORK  [conn65] received client metadata from 127.0.0.1:38118 conn65: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-1", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:07:59.545+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38120 #66 (22 connections now open)
2017-12-05T16:07:59.545+0900 I NETWORK  [conn66] received client metadata from 127.0.0.1:38120 conn66: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-2", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:08:01.526+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38142 #67 (23 connections now open)
2017-12-05T16:08:01.527+0900 I NETWORK  [conn67] received client metadata from 127.0.0.1:38142 conn67: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-3", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:08:56.158+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38604 #68 (24 connections now open)
2017-12-05T16:08:56.159+0900 I NETWORK  [conn68] received client metadata from 127.0.0.1:38604 conn68: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:13:44.774+0900 I -        [conn63] end connection 127.0.0.1:36578 (24 connections now open)
2017-12-05T16:13:57.881+0900 I -        [conn67] end connection 127.0.0.1:38142 (23 connections now open)
2017-12-05T16:17:46.253+0900 I -        [conn65] end connection 127.0.0.1:38118 (22 connections now open)
2017-12-05T16:17:47.372+0900 I -        [conn66] end connection 127.0.0.1:38120 (21 connections now open)
2017-12-05T16:18:40.688+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43394 #69 (21 connections now open)
2017-12-05T16:18:40.689+0900 I NETWORK  [conn69] received client metadata from 127.0.0.1:43394 conn69: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-3", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:19:07.407+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43626 #70 (22 connections now open)
2017-12-05T16:19:07.407+0900 I NETWORK  [conn70] received client metadata from 127.0.0.1:43626 conn70: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-0", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:19:13.799+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43692 #71 (23 connections now open)
2017-12-05T16:19:13.800+0900 I NETWORK  [conn71] received client metadata from 127.0.0.1:43692 conn71: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-1", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:19:16.300+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43712 #72 (24 connections now open)
2017-12-05T16:19:16.301+0900 I NETWORK  [conn72] received client metadata from 127.0.0.1:43712 conn72: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-2", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:27:25.761+0900 I -        [conn69] end connection 127.0.0.1:43394 (24 connections now open)
2017-12-05T16:27:27.600+0900 I -        [conn70] end connection 127.0.0.1:43626 (23 connections now open)
2017-12-05T16:27:27.604+0900 I -        [conn71] end connection 127.0.0.1:43692 (22 connections now open)
2017-12-05T16:27:29.270+0900 I -        [conn72] end connection 127.0.0.1:43712 (21 connections now open)
2017-12-05T16:28:53.069+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-05T16:28:53.070+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-05T17:08:56.199+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 2373 secs, remote host 127.0.0.1:57017)
2017-12-05T17:08:56.199+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 2373 secs, remote host 127.0.0.1:57042)
2017-12-05T17:08:56.199+0900 I COMMAND  [conn60] command local.oplog.rs command: getMore { getMore: 38240591073, collection: "oplog.rs", maxTimeMS: 5000, term: 7, lastKnownCommittedOpTime: { ts: Timestamp 1512458979000|1, t: 7 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512455885000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 7, readConcern: { afterOpTime: { ts: Timestamp 1512455885000|1, t: 6 } } } planSummary: COLLSCAN cursorid:38240591073 keysExamined:0 docsExamined:0 numYields:0 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 4 } }, Database: { acquireCount: { r: 2 } }, oplog: { acquireCount: { r: 2 } } } protocol:op_command 2351949ms
2017-12-05T17:08:56.199+0900 I REPL     [ReplicationExecutor] can't see a majority of the set, relinquishing primary
2017-12-05T17:08:56.199+0900 I REPL     [ReplicationExecutor] Stepping down from primary in response to heartbeat
2017-12-05T17:08:56.200+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state SECONDARY
2017-12-05T17:08:56.200+0900 I REPL     [ReplicationExecutor] Member localhost:47019 is now in state SECONDARY
2017-12-05T17:08:56.223+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57042 as failed :: caused by :: NotMaster: not master
2017-12-05T17:08:56.224+0900 I REPL     [replExecDBWorker-2] transition to SECONDARY
2017-12-05T17:08:56.224+0900 I NETWORK  [replExecDBWorker-2] legacy transport layer closing all connections
2017-12-05T17:08:56.224+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 64
2017-12-05T17:08:56.224+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 61
2017-12-05T17:08:56.224+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 60
2017-12-05T17:08:56.224+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 58
2017-12-05T17:08:56.224+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 2
2017-12-05T17:08:56.224+0900 I -        [conn48] end connection 127.0.0.1:54176 (20 connections now open)
2017-12-05T17:08:56.225+0900 I -        [conn46] end connection 127.0.0.1:54026 (19 connections now open)
2017-12-05T17:08:56.225+0900 I -        [conn68] end connection 127.0.0.1:38604 (18 connections now open)
2017-12-05T17:08:56.224+0900 I -        [conn62] end connection 127.0.0.1:36510 (20 connections now open)
2017-12-05T17:08:56.226+0900 I -        [conn51] end connection 127.0.0.1:54200 (17 connections now open)
2017-12-05T17:08:56.226+0900 I -        [conn50] end connection 127.0.0.1:54190 (15 connections now open)
2017-12-05T17:08:56.226+0900 I -        [conn54] end connection 127.0.0.1:54224 (14 connections now open)
2017-12-05T17:08:56.226+0900 I -        [conn52] end connection 127.0.0.1:54216 (13 connections now open)
2017-12-05T17:08:56.226+0900 I -        [conn47] end connection 127.0.0.1:54034 (12 connections now open)
2017-12-05T17:08:56.224+0900 I -        [conn55] end connection 127.0.0.1:54232 (20 connections now open)
2017-12-05T17:08:56.226+0900 I NETWORK  [replSetDistLockPinger] Successfully connected to localhost:57042 (1 connections now open to localhost:57042 with a 5 second timeout)
2017-12-05T17:08:56.229+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-05T17:08:56.233+0900 I -        [conn57] end connection 127.0.0.1:54242 (10 connections now open)
2017-12-05T17:08:56.233+0900 I -        [conn49] end connection 127.0.0.1:54186 (10 connections now open)
2017-12-05T17:08:56.233+0900 I -        [conn56] end connection 127.0.0.1:54236 (10 connections now open)
2017-12-05T17:08:56.234+0900 I -        [conn53] end connection 127.0.0.1:54222 (10 connections now open)
2017-12-05T17:08:56.234+0900 I -        [conn45] end connection 127.0.0.1:54018 (10 connections now open)
2017-12-05T17:08:56.732+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-05T17:08:57.234+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-05T17:08:57.762+0900 I -        [conn64] end connection 127.0.0.1:37106 (5 connections now open)
2017-12-05T17:09:01.249+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48044 #73 (5 connections now open)
2017-12-05T17:09:01.250+0900 I NETWORK  [conn73] received client metadata from 127.0.0.1:48044 conn73: { driver: { name: "NetworkInterfaceASIO-ShardRegistry", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:02.024+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48064 #74 (6 connections now open)
2017-12-05T17:09:02.024+0900 I NETWORK  [conn74] received client metadata from 127.0.0.1:48064 conn74: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:02.030+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48070 #75 (7 connections now open)
2017-12-05T17:09:02.030+0900 I NETWORK  [conn75] received client metadata from 127.0.0.1:48070 conn75: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:02.033+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48076 #76 (8 connections now open)
2017-12-05T17:09:02.033+0900 I NETWORK  [conn76] received client metadata from 127.0.0.1:48076 conn76: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:02.034+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48078 #77 (9 connections now open)
2017-12-05T17:09:02.034+0900 I NETWORK  [conn77] received client metadata from 127.0.0.1:48078 conn77: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:02.360+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48086 #78 (10 connections now open)
2017-12-05T17:09:02.361+0900 I NETWORK  [conn78] received client metadata from 127.0.0.1:48086 conn78: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:02.364+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48092 #79 (11 connections now open)
2017-12-05T17:09:02.365+0900 I NETWORK  [conn79] received client metadata from 127.0.0.1:48092 conn79: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:02.410+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48096 #80 (12 connections now open)
2017-12-05T17:09:02.411+0900 I NETWORK  [conn80] received client metadata from 127.0.0.1:48096 conn80: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:02.412+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-05T17:09:02.413+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48100 #81 (13 connections now open)
2017-12-05T17:09:02.414+0900 I NETWORK  [conn81] received client metadata from 127.0.0.1:48100 conn81: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:02.414+0900 W NETWORK  [ReplicaSetMonitor-TaskExecutor-0] No primary detected for set s2
2017-12-05T17:09:02.416+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48104 #82 (14 connections now open)
2017-12-05T17:09:02.416+0900 I NETWORK  [conn82] received client metadata from 127.0.0.1:48104 conn82: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:02.417+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 6 secs, remote host 127.0.0.1:47018)
2017-12-05T17:09:02.420+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48106 #83 (15 connections now open)
2017-12-05T17:09:02.420+0900 I NETWORK  [conn83] received client metadata from 127.0.0.1:48106 conn83: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:02.420+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47018 (1 connections now open to localhost:47018 with a 5 second timeout)
2017-12-05T17:09:02.422+0900 W NETWORK  [ReplicaSetMonitor-TaskExecutor-0] No primary detected for set s1
2017-12-05T17:09:06.312+0900 I REPL     [ReplicationExecutor] Starting an election, since we've seen no PRIMARY in the past 10000ms
2017-12-05T17:09:06.312+0900 I REPL     [ReplicationExecutor] conducting a dry run election to see if we could be elected
2017-12-05T17:09:06.312+0900 I REPL     [ReplicationExecutor] VoteRequester(term 7 dry run) received a yes vote from localhost:47017; response message: { term: 7, voteGranted: true, reason: "", ok: 1.0 }
2017-12-05T17:09:06.313+0900 I REPL     [ReplicationExecutor] dry election run succeeded, running for election
2017-12-05T17:09:06.316+0900 I REPL     [ReplicationExecutor] VoteRequester(term 8) received a yes vote from localhost:47017; response message: { term: 8, voteGranted: true, reason: "", ok: 1.0 }
2017-12-05T17:09:06.316+0900 I REPL     [ReplicationExecutor] election succeeded, assuming primary role in term 8
2017-12-05T17:09:06.316+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending connection to host localhost:47019 due to bad connection status; 0 connections to that host remain open
2017-12-05T17:09:06.316+0900 I REPL     [ReplicationExecutor] transition to PRIMARY
2017-12-05T17:09:06.317+0900 I REPL     [ReplicationExecutor] Entering primary catch-up mode.
2017-12-05T17:09:06.317+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47019
2017-12-05T17:09:06.318+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47019, took 1ms (1 connections now open to localhost:47019)
2017-12-05T17:09:06.318+0900 I REPL     [ReplicationExecutor] Caught up to the latest optime known via heartbeats after becoming primary.
2017-12-05T17:09:06.319+0900 I REPL     [ReplicationExecutor] Exited primary catch-up mode.
2017-12-05T17:09:08.183+0900 I REPL     [rsSync] transition to primary complete; database writes are now permitted
2017-12-05T17:09:10.468+0900 I -        [conn61] end connection 127.0.0.1:35984 (15 connections now open)
2017-12-05T17:09:11.247+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48124 #84 (15 connections now open)
2017-12-05T17:09:11.247+0900 I NETWORK  [conn84] received client metadata from 127.0.0.1:48124 conn84: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:11.248+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48126 #85 (16 connections now open)
2017-12-05T17:09:11.249+0900 I NETWORK  [conn85] received client metadata from 127.0.0.1:48126 conn85: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:11.251+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48128 #86 (17 connections now open)
2017-12-05T17:09:11.251+0900 I NETWORK  [conn86] received client metadata from 127.0.0.1:48128 conn86: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:19.192+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48136 #87 (18 connections now open)
2017-12-05T17:09:19.192+0900 I NETWORK  [conn87] received client metadata from 127.0.0.1:48136 conn87: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:19.324+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48140 #88 (19 connections now open)
2017-12-05T17:09:19.324+0900 I NETWORK  [conn88] received client metadata from 127.0.0.1:48140 conn88: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:19.817+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48144 #89 (20 connections now open)
2017-12-05T17:09:19.817+0900 I NETWORK  [conn89] received client metadata from 127.0.0.1:48144 conn89: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:31.588+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T17:09:31.590+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T17:24:01.621+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T17:24:01.623+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T17:38:31.655+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-05T17:38:31.656+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (2 connections now open to localhost:57041)
2017-12-05T17:39:31.657+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Ending idle connection to host localhost:57041 because the pool meets constraints; 1 connections to that host remain open
2017-12-05T17:50:02.300+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T17:50:02.304+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-05T18:05:02.338+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T18:05:02.339+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-05T18:33:32.394+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-05T18:33:32.397+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 3ms (2 connections now open to localhost:57041)
2017-12-05T18:34:32.398+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Ending idle connection to host localhost:57041 because the pool meets constraints; 1 connections to that host remain open
2017-12-05T19:00:32.462+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T19:00:32.464+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T19:35:02.552+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T19:35:02.553+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T19:44:03.458+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56892 #90 (21 connections now open)
2017-12-05T19:44:03.459+0900 I NETWORK  [conn90] received client metadata from 127.0.0.1:56892 conn90: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T19:45:03.459+0900 I -        [conn86] end connection 127.0.0.1:48128 (21 connections now open)
2017-12-05T19:57:32.604+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T19:57:32.607+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 3ms (1 connections now open to localhost:57042)
2017-12-05T20:25:32.668+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T20:25:32.673+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 5ms (1 connections now open to localhost:57040)
2017-12-05T20:46:02.722+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T20:46:02.723+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-05T21:01:02.752+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T21:01:02.756+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 4ms (1 connections now open to localhost:57040)
2017-12-05T21:12:02.785+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T21:12:02.788+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 3ms (1 connections now open to localhost:57040)
2017-12-05T21:22:02.811+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T21:22:02.812+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T22:04:32.910+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T22:04:32.912+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T22:42:02.990+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T22:42:02.992+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T23:07:03.048+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T23:07:03.049+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T23:33:03.104+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T23:33:03.106+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T23:39:33.120+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T23:39:33.122+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T23:45:03.872+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:32854 #91 (21 connections now open)
2017-12-05T23:45:03.872+0900 I NETWORK  [conn91] received client metadata from 127.0.0.1:32854 conn91: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T23:46:03.872+0900 I -        [conn90] end connection 127.0.0.1:56892 (21 connections now open)
2017-12-05T23:50:03.149+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T23:50:03.151+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T00:30:33.252+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T00:30:33.253+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-06T00:35:38.966+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:33100 #92 (21 connections now open)
2017-12-06T00:35:38.967+0900 I NETWORK  [conn92] received client metadata from 127.0.0.1:33100 conn92: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T00:36:38.967+0900 I -        [conn84] end connection 127.0.0.1:48124 (21 connections now open)
2017-12-06T00:37:43.970+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:33104 #93 (21 connections now open)
2017-12-06T00:37:43.970+0900 I NETWORK  [conn93] received client metadata from 127.0.0.1:33104 conn93: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T00:38:43.973+0900 I -        [conn91] end connection 127.0.0.1:32854 (21 connections now open)
2017-12-06T00:46:33.294+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T00:46:33.298+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-06T01:04:03.339+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T01:04:03.340+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T01:07:03.347+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T01:07:03.349+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-06T01:16:34.052+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:33336 #94 (21 connections now open)
2017-12-06T01:16:34.053+0900 I NETWORK  [conn94] received client metadata from 127.0.0.1:33336 conn94: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T01:17:34.052+0900 I -        [conn92] end connection 127.0.0.1:33100 (21 connections now open)
2017-12-06T01:21:19.057+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:33358 #95 (21 connections now open)
2017-12-06T01:21:19.058+0900 I NETWORK  [conn95] received client metadata from 127.0.0.1:33358 conn95: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T01:22:19.057+0900 I -        [conn94] end connection 127.0.0.1:33336 (21 connections now open)
2017-12-06T02:10:03.509+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T02:10:03.511+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-06T02:16:03.524+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T02:16:03.526+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-06T02:41:03.593+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T02:41:03.595+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-06T03:04:03.652+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T03:04:03.654+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-06T03:07:04.260+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:33910 #96 (21 connections now open)
2017-12-06T03:07:04.261+0900 I NETWORK  [conn96] received client metadata from 127.0.0.1:33910 conn96: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T03:08:04.261+0900 I -        [conn93] end connection 127.0.0.1:33104 (21 connections now open)
2017-12-06T03:12:33.675+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T03:12:33.676+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T03:41:03.746+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T03:41:03.746+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 0ms (1 connections now open to localhost:57040)
2017-12-06T04:57:03.934+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T04:57:03.936+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-06T05:16:24.514+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:34518 #97 (21 connections now open)
2017-12-06T05:16:24.515+0900 I NETWORK  [conn97] received client metadata from 127.0.0.1:34518 conn97: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T05:17:24.514+0900 I -        [conn96] end connection 127.0.0.1:33910 (21 connections now open)
2017-12-06T05:18:33.991+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T05:18:33.993+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-06T05:42:34.565+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:34682 #98 (21 connections now open)
2017-12-06T05:42:34.566+0900 I NETWORK  [conn98] received client metadata from 127.0.0.1:34682 conn98: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T05:43:34.566+0900 I -        [conn95] end connection 127.0.0.1:33358 (21 connections now open)
2017-12-06T06:10:04.143+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T06:10:04.145+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-06T06:17:54.634+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:34836 #99 (21 connections now open)
2017-12-06T06:17:54.634+0900 I NETWORK  [conn99] received client metadata from 127.0.0.1:34836 conn99: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T06:18:54.634+0900 I -        [conn97] end connection 127.0.0.1:34518 (21 connections now open)
2017-12-06T06:47:34.244+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T06:47:34.245+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T07:43:04.370+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T07:43:04.374+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 4ms (1 connections now open to localhost:57040)
2017-12-06T07:48:34.384+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T07:48:34.386+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-06T18:08:08.175+0900 I -        [conn98] end connection 127.0.0.1:34682 (20 connections now open)
2017-12-06T18:08:08.176+0900 I -        [conn99] end connection 127.0.0.1:34836 (19 connections now open)
2017-12-06T18:08:08.178+0900 I COMMAND  [conn60] command local.oplog.rs command: getMore { getMore: 37225257907, collection: "oplog.rs", maxTimeMS: 5000, term: 8, lastKnownCommittedOpTime: { ts: Timestamp 1512515099000|1, t: 8 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512461336000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 8, readConcern: { afterOpTime: { ts: Timestamp 1512461336000|1, t: 7 } } } planSummary: COLLSCAN cursorid:37225257907 keysExamined:0 docsExamined:0 numYields:1 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 6 } }, Database: { acquireCount: { r: 3 } }, oplog: { acquireCount: { r: 3 } } } protocol:op_command 36188328ms
2017-12-06T18:08:08.190+0900 I -        [conn60] end connection 127.0.0.1:35802 (18 connections now open)
2017-12-06T18:08:08.182+0900 I REPL     [ReplicationExecutor] can't see a majority of the set, relinquishing primary
2017-12-06T18:08:08.191+0900 I REPL     [ReplicationExecutor] Stepping down from primary in response to heartbeat
2017-12-06T18:08:08.198+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state SECONDARY
2017-12-06T18:08:08.198+0900 I COMMAND  [ftdc] serverStatus was very slow: { after basic: 0, after asserts: 0, after backgroundFlushing: 0, after connections: 0, after dur: 0, after extra_info: 0, after globalLock: 0, after locks: 0, after network: 0, after opLatencies: 0, after opcounters: 0, after opcountersRepl: 0, after repl: 36183599, after security: 36183599, after storageEngine: 36183599, after tcmalloc: 36183599, after wiredTiger: 36183599, at end: 36183599 }
2017-12-06T18:08:08.202+0900 I COMMAND  [conn85] command local.oplog.rs command: getMore { getMore: 34696671571, collection: "oplog.rs", maxTimeMS: 5000, term: 8, lastKnownCommittedOpTime: { ts: Timestamp 1512515099000|1, t: 8 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512461336000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 8, readConcern: { afterOpTime: { ts: Timestamp 1512461336000|1, t: 7 } } } planSummary: COLLSCAN cursorid:34696671571 keysExamined:0 docsExamined:0 numYields:0 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 4 } }, Database: { acquireCount: { r: 2 } }, oplog: { acquireCount: { r: 2 } } } protocol:op_command 36188352ms
2017-12-06T18:08:08.202+0900 I -        [conn85] end connection 127.0.0.1:48126 (17 connections now open)
2017-12-06T18:08:08.205+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35374 #100 (17 connections now open)
2017-12-06T18:08:08.205+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35376 #101 (18 connections now open)
2017-12-06T18:08:08.205+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35378 #102 (19 connections now open)
2017-12-06T18:08:08.206+0900 I NETWORK  [conn102] received client metadata from 127.0.0.1:35378 conn102: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:08.206+0900 I NETWORK  [conn101] received client metadata from 127.0.0.1:35376 conn101: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:08.206+0900 I NETWORK  [conn100] received client metadata from 127.0.0.1:35374 conn100: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:08.210+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35392 #103 (20 connections now open)
2017-12-06T18:08:08.210+0900 I NETWORK  [conn103] received client metadata from 127.0.0.1:35392 conn103: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:08.210+0900 I REPL     [replExecDBWorker-1] transition to SECONDARY
2017-12-06T18:08:08.216+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35394 #104 (21 connections now open)
2017-12-06T18:08:08.216+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35396 #105 (22 connections now open)
2017-12-06T18:08:08.218+0900 I NETWORK  [replExecDBWorker-1] legacy transport layer closing all connections
2017-12-06T18:08:08.218+0900 I NETWORK  [replExecDBWorker-1] Skip closing connection for connection # 103
2017-12-06T18:08:08.218+0900 I NETWORK  [replExecDBWorker-1] Skip closing connection for connection # 102
2017-12-06T18:08:08.218+0900 I NETWORK  [replExecDBWorker-1] Skip closing connection for connection # 101
2017-12-06T18:08:08.218+0900 I NETWORK  [replExecDBWorker-1] Skip closing connection for connection # 100
2017-12-06T18:08:08.218+0900 I NETWORK  [replExecDBWorker-1] Skip closing connection for connection # 73
2017-12-06T18:08:08.218+0900 I NETWORK  [replExecDBWorker-1] Skip closing connection for connection # 58
2017-12-06T18:08:08.218+0900 I NETWORK  [replExecDBWorker-1] Skip closing connection for connection # 2
2017-12-06T18:08:08.218+0900 I -        [conn89] end connection 127.0.0.1:48144 (22 connections now open)
2017-12-06T18:08:08.219+0900 I -        [conn78] end connection 127.0.0.1:48086 (21 connections now open)
2017-12-06T18:08:08.219+0900 I -        [conn87] end connection 127.0.0.1:48136 (20 connections now open)
2017-12-06T18:08:08.219+0900 I -        [conn83] end connection 127.0.0.1:48106 (19 connections now open)
2017-12-06T18:08:08.219+0900 I -        [conn76] end connection 127.0.0.1:48076 (18 connections now open)
2017-12-06T18:08:08.219+0900 I -        [conn74] end connection 127.0.0.1:48064 (17 connections now open)
2017-12-06T18:08:08.220+0900 I -        [conn80] end connection 127.0.0.1:48096 (16 connections now open)
2017-12-06T18:08:08.220+0900 I -        [conn79] end connection 127.0.0.1:48092 (15 connections now open)
2017-12-06T18:08:08.220+0900 I -        [conn75] end connection 127.0.0.1:48070 (14 connections now open)
2017-12-06T18:08:08.220+0900 I -        [conn88] end connection 127.0.0.1:48140 (13 connections now open)
2017-12-06T18:08:08.222+0900 I -        [conn105] end connection 127.0.0.1:35396 (12 connections now open)
2017-12-06T18:08:08.222+0900 I -        [conn104] end connection 127.0.0.1:35394 (11 connections now open)
2017-12-06T18:08:08.239+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35402 #106 (8 connections now open)
2017-12-06T18:08:08.239+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35404 #107 (9 connections now open)
2017-12-06T18:08:08.239+0900 I NETWORK  [conn107] received client metadata from 127.0.0.1:35404 conn107: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:08.239+0900 I NETWORK  [conn106] received client metadata from 127.0.0.1:35402 conn106: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:08.254+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Detected bad connection created at 1512461342418045 microSec, clearing pool for localhost:47018 of 0 connections
2017-12-06T18:08:08.257+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57041 as failed :: caused by :: NotMaster: not master
2017-12-06T18:08:08.257+0900 I REPL     [ReplicationExecutor] Member localhost:47019 is now in state SECONDARY
2017-12-06T18:08:08.258+0900 I -        [conn77] AssertionException handling request, closing client connection: 172 Operation attempted on a closed transport Session.
2017-12-06T18:08:08.258+0900 I -        [conn77] end connection 127.0.0.1:48078 (12 connections now open)
2017-12-06T18:08:08.259+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Dropping all pooled connections to localhost:47018(with timeout of 5 seconds)
2017-12-06T18:08:08.259+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Ending connection to host localhost:47018(with timeout of 5 seconds) due to bad connection status; 0 connections to that host remain open
2017-12-06T18:08:08.259+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Marking host localhost:47018 as failed :: caused by :: HostUnreachable: network error while attempting to run command 'ismaster' on host 'localhost:47018' 
2017-12-06T18:08:08.259+0900 I NETWORK  [replSetDistLockPinger] Detected bad connection created at 1512455453783220 microSec, clearing pool for localhost:57041 of 0 connections
2017-12-06T18:08:08.269+0900 I NETWORK  [replSetDistLockPinger] Dropping all pooled connections to localhost:57041(with timeout of 5 seconds)
2017-12-06T18:08:08.269+0900 I NETWORK  [replSetDistLockPinger] Ending connection to host localhost:57041(with timeout of 5 seconds) due to bad connection status; 0 connections to that host remain open
2017-12-06T18:08:08.266+0900 I -        [conn81] AssertionException handling request, closing client connection: 172 Operation attempted on a closed transport Session.
2017-12-06T18:08:08.270+0900 I -        [conn81] end connection 127.0.0.1:48100 (11 connections now open)
2017-12-06T18:08:08.266+0900 I -        [conn58] end connection 127.0.0.1:60154 (11 connections now open)
2017-12-06T18:08:08.266+0900 I -        [conn82] AssertionException handling request, closing client connection: 172 Operation attempted on a closed transport Session.
2017-12-06T18:08:08.271+0900 I -        [conn82] end connection 127.0.0.1:48104 (9 connections now open)
2017-12-06T18:08:08.270+0900 W NETWORK  [ReplicaSetMonitor-TaskExecutor-0] No primary detected for set s1
2017-12-06T18:08:08.272+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:08.772+0900 I NETWORK  [replSetDistLockPinger] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-06T18:08:08.772+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:09.274+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:09.776+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:10.277+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:10.779+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:11.281+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:11.782+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:12.283+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:12.785+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:13.258+0900 I REPL     [ReplicationExecutor] Member localhost:47019 is now in state PRIMARY
2017-12-06T18:08:13.286+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:13.787+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:14.180+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:47019
2017-12-06T18:08:14.180+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-06T18:08:14.181+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 1ms (1 connections now open to localhost:47019)
2017-12-06T18:08:14.192+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-06T18:08:14.193+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 1ms (2 connections now open to localhost:47019)
2017-12-06T18:08:14.288+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:14.789+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:15.291+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:15.791+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:16.292+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:16.793+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:17.294+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:17.795+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:18.296+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:18.798+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:19.298+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:19.800+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-06T18:08:20.736+0900 I -        [conn73] end connection 127.0.0.1:48044 (8 connections now open)
2017-12-06T18:08:24.268+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35472 #108 (8 connections now open)
2017-12-06T18:08:24.268+0900 I NETWORK  [conn108] received client metadata from 127.0.0.1:35472 conn108: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:24.443+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35482 #109 (9 connections now open)
2017-12-06T18:08:24.443+0900 I NETWORK  [conn109] received client metadata from 127.0.0.1:35482 conn109: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:24.522+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35490 #110 (10 connections now open)
2017-12-06T18:08:24.522+0900 I NETWORK  [conn110] received client metadata from 127.0.0.1:35490 conn110: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:25.101+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35498 #111 (11 connections now open)
2017-12-06T18:08:25.101+0900 I NETWORK  [conn111] received client metadata from 127.0.0.1:35498 conn111: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.202+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35594 #112 (12 connections now open)
2017-12-06T18:08:38.202+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35592 #113 (13 connections now open)
2017-12-06T18:08:38.202+0900 I NETWORK  [conn112] received client metadata from 127.0.0.1:35594 conn112: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.202+0900 I NETWORK  [conn113] received client metadata from 127.0.0.1:35592 conn113: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.209+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35602 #114 (14 connections now open)
2017-12-06T18:08:38.209+0900 I NETWORK  [conn114] received client metadata from 127.0.0.1:35602 conn114: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.217+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 30 secs, remote host 127.0.0.1:37018)
2017-12-06T18:08:38.219+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37018 (1 connections now open to localhost:37018 with a 5 second timeout)
2017-12-06T18:08:38.223+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 30 secs, remote host 127.0.0.1:57018)
2017-12-06T18:08:38.224+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57018 (1 connections now open to localhost:57018 with a 5 second timeout)
2017-12-06T18:08:38.259+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35620 #115 (15 connections now open)
2017-12-06T18:08:38.259+0900 I NETWORK  [conn115] received client metadata from 127.0.0.1:35620 conn115: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.260+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35622 #116 (16 connections now open)
2017-12-06T18:08:38.260+0900 I NETWORK  [conn116] received client metadata from 127.0.0.1:35622 conn116: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.261+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35624 #117 (17 connections now open)
2017-12-06T18:08:38.261+0900 I NETWORK  [conn117] received client metadata from 127.0.0.1:35624 conn117: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.268+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35632 #118 (18 connections now open)
2017-12-06T18:08:38.270+0900 I NETWORK  [conn118] received client metadata from 127.0.0.1:35632 conn118: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.273+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35648 #119 (19 connections now open)
2017-12-06T18:08:38.273+0900 I NETWORK  [conn119] received client metadata from 127.0.0.1:35648 conn119: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.275+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:35652 #120 (20 connections now open)
2017-12-06T18:08:38.275+0900 I NETWORK  [conn120] received client metadata from 127.0.0.1:35652 conn120: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.279+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47018 (1 connections now open to localhost:47018 with a 5 second timeout)
2017-12-06T18:09:08.259+0900 I -        [conn102] end connection 127.0.0.1:35378 (20 connections now open)
2017-12-06T18:09:08.268+0900 I -        [conn103] end connection 127.0.0.1:35392 (19 connections now open)
2017-12-06T18:09:08.277+0900 I -        [conn107] end connection 127.0.0.1:35404 (18 connections now open)
2017-12-06T18:10:36.281+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T18:10:36.283+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-06T18:13:08.272+0900 I -        [conn101] end connection 127.0.0.1:35376 (17 connections now open)
2017-12-06T18:13:08.277+0900 I -        [conn106] end connection 127.0.0.1:35402 (16 connections now open)
2017-12-06T18:17:36.292+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T18:17:36.292+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 0ms (1 connections now open to localhost:57042)
2017-12-06T18:42:06.774+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T18:42:06.776+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-06T18:52:06.798+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T18:52:06.800+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-06T19:21:06.875+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T19:21:06.877+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-06T19:36:06.914+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T19:36:06.917+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 3ms (1 connections now open to localhost:57042)
2017-12-06T19:48:06.945+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T19:48:06.947+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-06T20:15:37.028+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T20:15:37.029+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-06T20:40:04.555+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-06T20:40:04.558+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 3ms (3 connections now open to localhost:47019)
2017-12-06T20:40:37.072+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T20:40:37.073+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-06T20:41:04.557+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47019 because the pool meets constraints; 2 connections to that host remain open
2017-12-06T21:14:37.145+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T21:14:37.146+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T21:42:37.213+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T21:42:37.215+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-06T21:59:34.672+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-06T21:59:34.673+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 2ms (3 connections now open to localhost:47019)
2017-12-06T22:00:34.673+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47019 because the pool meets constraints; 2 connections to that host remain open
2017-12-06T22:03:07.259+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T22:03:07.260+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T22:45:37.359+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T22:45:37.365+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 6ms (1 connections now open to localhost:57041)
2017-12-06T23:02:37.416+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T23:02:37.418+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-06T23:04:07.421+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T23:04:07.423+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-06T23:41:37.502+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T23:41:37.504+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-06T23:53:07.533+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T23:53:07.533+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-07T00:08:37.573+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T00:08:37.577+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-07T00:21:07.601+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T00:21:07.601+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 0ms (1 connections now open to localhost:57042)
2017-12-07T00:44:07.647+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T00:44:07.648+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-07T00:58:37.682+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-07T00:58:37.683+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-07T01:11:07.715+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-07T01:11:07.717+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 3ms (1 connections now open to localhost:57041)
2017-12-07T01:31:37.764+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-07T01:31:37.766+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-07T02:26:37.911+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T02:26:37.913+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-07T04:03:38.177+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-07T04:03:38.179+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-07T05:05:38.336+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T05:05:38.336+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 0ms (1 connections now open to localhost:57042)
2017-12-07T05:10:25.531+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-07T05:10:25.533+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 2ms (3 connections now open to localhost:47019)
2017-12-07T05:11:25.531+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47019 because the pool meets constraints; 2 connections to that host remain open
2017-12-07T06:05:08.490+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T06:05:08.491+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-07T06:13:38.509+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T06:13:38.511+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-07T06:31:35.695+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47019
2017-12-07T06:31:35.696+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47019, took 2ms (3 connections now open to localhost:47019)
2017-12-07T06:32:35.696+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47019 because the pool meets constraints; 2 connections to that host remain open
